doc/news/changes/incompatibilities/20200727DavidWells

```diff
@@ -0,0 +1,5 @@
+Removed: The IB6 delta function (i.e., lagrangian_ib_6_delta) has been removed
```

Comment:
This comment suggests that we don't support the IB_6 kernel function, but we do -- just not though this particular interface.

Reply:
Good point. I clarified the comment.